### DISCUSSION TITLE
Add Azure Linux runner image

### DISCUSSION THREAD

**Discussion by cataggar** [actions/virtual-environments#8987] [Ideas]
Add an Azure Linux (aka Mariner) runner image.
- [aka.ms/cbl-mariner](https://aka.ms/cbl-mariner)
- MS internal: - [aka.ms/mariner](https://aka.ms/mariner)

**Comment by erik-bershel** [comment]
@mikhailkoliada do we have plans to do it?

**Comment by mikhailkoliada** [comment]
@cataggar hey! There are no plans to add any new linux distributions at the moment.
